Adyen/Helpers/UIViewHelpers.swift

```diff
@@ -20,9 +20,18 @@ extension AdyenScope where Base: UIView {
 
         return image
     }
+    
+    /// :nodoc:
+    public func hide(hidden: Bool, animated: Bool) {
+        if animated {
+            hideWithAnimation(hidden)
+        } else {
+            hideWithoutAnimation(hidden)
+        }
+    }
 
     /// :nodoc:
-    public func hideWithAnimation(_ hidden: Bool) {
+    private func hideWithAnimation(_ hidden: Bool) {
```

Comment:
If you set `withDuration: animated  : 0.35 ? 0` - will get similar effect :D
This two methods is also fine

Reply:
yeah i used to do it like that. but `hideWithAnimatoon` has a lot of logic, i thought it would be cleaner to have a small function with no animation

Reply:
👍